docs/website/docs/reference/installation.md

```diff
@@ -23,9 +23,12 @@ If you have a different Python version installed or are missing pip, follow the
 You can install Python 3.10 with `apt`.
 
 ```sh
+sudo add-apt-repository ppa:deadsnakes/ppa
+sudo apt install -y software-properties-common
 sudo apt update
 sudo apt install python3.10
 sudo apt install python3.10-venv
+sudo apt install pip
```

Comment:
I think pip should be installed automatically together with python, is that not the case?

Reply:
Not always. And the main issue was that only the windows instructions installed pip explicitly.